core-ui/src/components/Extensibility/ds/widgets-form/KeyValuePairRenderer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { TransTitle } from '@ui-schema/ui-schema/Translate/TransTitle';
+
+import { ResourceForm } from 'shared/ResourceForm';
+import { KeyValueField } from 'shared/ResourceForm/fields';
+
+export function KeyValuePairRenderer({
+  storeKeys,
+  schema,
+  resource,
+  setResource,
+}) {
+  const propertyPath = `$.${storeKeys.toJS().join('.')}`;
+
+  return (
+    <ResourceForm.Wrapper resource={resource} setResource={setResource}>
```

Comment:
I'd consider using the ui-schema `value`/`onChange` instead of using `ResourceForm.Wrapper` - and connect them to `value`/`setValue` properties of `KeyValueField` (similar to, for example, `StringRenderer`)
(but I'll accept a "we can try changing it in another PR" answer)

Reply:
It would require changes in KeyValueField which is already used everywhere so maybe in next PR